### PR TITLE
Get Collection Stats too from db

### PR DESCRIPTION
**Pull request by Demfier**
Hello there,

Currently no collection metrics like ```insert_calls```, ```update_calls``` etc are being sent. I think they could be pretty useful too. I've added the ```collection_stats``` type in ```types.db``` too.

Hope you find it useful.

Cheers!

### CODE REVIEW

**Comment by Demfier** [comment]
@sebest can you suggest anything for improvements in this PR?

**Comment by sebest** [comment]
Hi Demfier,
It looks good to me, but I don't have a way to test it anymore as I'm no more using MongoDB.
So I'm not sure what would be the best way to proceed.

**Comment by Demfier** [comment]
@sebest I've setup a grafana dashboard for my MongoDB server and I am getting different kinds of collection-wise metrics like ```insert_calls```, ```update_calls``` etc. there.

**Comment by Demfier** [comment]
@sebest still this PR out of consideration?

File: mongodb.py [hunk 1]
```diff
@@ -56,18 +56,18 @@ def do_server_status(self):
...
             self.submit('connections', 'totalCreated', server_status['connections']['totalCreated'])
 
-	# network
-	if 'network' in server_status:
-	    for t in ['bytesIn', 'bytesOut', 'numRequests']:
-                self.submit('bytes', t, server_status['network'][t])
+        # network
+        if 'network' in server_status:
+            for t in ['bytesIn', 'bytesOut', 'numRequests']:
+                      self.submit('bytes', t, server_status['network'][t])
```
Comment (sebest) [inline mongodb.py]: it seems that there is too much indentation here?!

**Comment by Demfier** [comment]
There were some misaligned indentations when I downloaded the raw script from github. I missed this one. Thanks pointing this out! Tell me if I missed anything else.

**Review by sebest — APPROVED** [review]
Thanx a lot for your contribution!

**Comment by Demfier** [comment]
It's my pleasure!

**Comment by Demfier** [comment]
@sebest anything else to be done?

**Review by devotences — APPROVED** [review]

**Comment by sebest** [comment]
@Demfier : sorry, I forgot to push the "merge" button.

**Comment by Demfier** [comment]
@sebest - Haha, it's alright.

**Pull request merged**